src/feed/views/journal_feed_view.py

```diff
@@ -27,9 +28,11 @@ class JournalFeedViewSet(BaseFeedView):
     Query Parameters:
     - publication_status: Filter by publication status
       Options:
-        - PREPRINT: Only show preprints
-        - PUBLISHED: Only show published papers
-        - ALL: Show all papers (default)
+        - PREPRINT: Only show preprints in ResearchHub journal
+        - PUBLISHED: Only show published papers in ResearchHub journal
+        - NON_JOURNAL: Only show papers that are submitted but not published to
```

Comment:
The name for this filter parameter is confusion. Would it be clearer to call it `SUBMITTED` or `NON_PUBLISHED` instead?